aws_common/test/sdk_utils/throttling_manager_test.cpp

```diff
@@ -109,19 +109,21 @@ void MakeCalls(ThrottledClient * throttled_client, std::chrono::milliseconds dur
  */
 TEST(ThrottlingManagerTest, multiThreadedClientThrottling)
 {
-  int milliseconds_to_run = 3500, sleep_duration_in_us = 150, max_tps = 125;
+  float milliseconds_to_run = 3500, max_tps = 125;
```

Comment:
Keep these defined as integers as they are used as integers in all but mathematical operations. Also make them "const" since they are not altered and therefore is easier to determine the flow of this test

Reply:
done.